src/test/java/org/cactoos/io/ZipTest.java

```diff
@@ -24,51 +24,80 @@
 
 package org.cactoos.io;
 
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.zip.ZipEntry;
-import java.util.zip.ZipInputStream;
-import org.hamcrest.MatcherAssert;
-import org.hamcrest.Matchers;
+import java.io.File;
+import java.io.OutputStream;
+import java.util.zip.ZipFile;
+import org.cactoos.Text;
+import org.cactoos.iterator.Mapped;
+import org.cactoos.list.ListOf;
+import org.cactoos.scalar.Sticky;
+import org.cactoos.text.Joined;
+import org.cactoos.text.TextOf;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
+import org.llorllale.cactoos.matchers.Assertion;
+import org.llorllale.cactoos.matchers.ScalarHasValue;
 
 /**
  * Test case for {@link Zip}.
  *
  * @since 0.29
  * @checkstyle JavadocMethodCheck (500 lines)
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
```

Comment:
@fanifieiev please remove this, it seems the only repeated literal is `abc`, `B.txt` and `A.txt`, so you should be able to allocate the `File` to a variable and refer to their name in the rest of the test.

Reply:
@victornoel Ok